content/docs/user-guide/networking/multiplayer/VersionMismatch.md

```diff
@@ -0,0 +1,36 @@
+---
+linkTitle: Version Mismatch
+title: Multiplayer Version Mismatch
+description: Learn how to detect multiplayer version mismatch in Open 3D Engine (O3DE).
+weight: 900
+---
+
+In order to keep the multiplayer simulation in sync, it's important that all connected multiplayer endpoints are running the same multiplayer version.
+
+For example, consider a server that is running a particular build of a multiplayer game. If an updated client connects with changes to its network components, the server may not know how to handle the updated network properties, or how to serialize network packets correctly.
+
+Servers and clients must be running the same version of all the multiplayer components (components which communicate to each other over the network); any differences may lead to unexpected behavior. 
```

Comment:
any differences or any difference?

Reply:
Oh, nice one... had to google! If the differences are countable you can use differences. But what I'm trying to say is that _even one_ difference would cause unexpected behavior.